pkgs/servers/code-server/default.nix

```diff
@@ -241,6 +241,9 @@ in stdenv.mkDerivation rec {
     prefetchYarnCache = lib.overrideDerivation yarnCache (d: {
       outputHash = lib.fakeSha256;
     });
+    # vscode-with-extensions compatibility
+    executableName = "code-server";
+    longName = "Visual Studio Code Server";
```

Comment:
Can you explain these lines?

Reply:
https://github.com/NixOS/nixpkgs/blob/7a7242db844fdb9e64ee2d7cf345febaebb07b24/pkgs/applications/editors/vscode/with-extensions.nix#L45

Reply:
Fair enough, thanks !